front/types/provider.ts

```diff
@@ -2,3 +2,15 @@ export type ProviderType = {
   providerId: string;
   config: string;
 };
+
+export type CredentialsType = {
```

Comment:
Can you delete from [here](https://github.com/dust-tt/dust/blob/d3e25aa2f72c4ae9bb76fd7fef4d10c9c23b3bc2/front/lib/providers.ts#L171-L181) 🙏

Reply:
uh already done I moved it from there :)

Reply:
Oh my bad missed that 🤦